Updates comment reference to non-deprecated class

JordanDTaylor opened the pull request:
System.Configuration.ConfigurationSettings.AppSettings was deprecated in 2.0.

# Fixes comment referencing deprecated class.

## Summary
This is a tiny documentation change.

`System.Configuration.ConfigurationSettings.AppSettings` was deprecated in .NET 2.0.  I modified the comment to suggest using `System.Configuration.ConfigurationManager.AppSettings`

JordanDTaylor committed:
Updates comment reference non-deprecated class 

System.Configuration.ConfigurationSettings.AppSettings was deprecated in 2.0.

rpetrusha reviewed (changes requested):
Thanks for contributing to the dotnet/docs repo and for making this correction, @JordanDTaylor. Not only is the ConfigurationSettings class deprecated, but the best practice when storing connection strings in configuration files is to use the \<connectionStrings> section. The ConfigurationManager.ConnectionStrings property can then be used to retrieve the connection string(s). Would you mind making an additional change by changing AppSettings  to ConnectionStrings? If you'd prefer, we can also make the change for you.

JordanDTaylor committed:
Changes AppSettings  to ConnectionStrings

JordanDTaylor commented:
Excellent point.  I have updated to ConnectionStrings.  Thanks!

rpetrusha commented:
Thanks for making the additional change, @JordanDTaylor. We'll merge your PR as soon as our internal build completes successfully. The update should appear on docs.microsoft.com in the next day or two.

mairaw commented:
We'd need to make the same change to the VB counterpart:
https://github.com/JordanDTaylor/docs/blob/patch-1/samples/snippets/visualbasic/VS_Snippets_ADO.NET/Classic%20WebData%20SqlConnection.SqlConnection%20Example/VB/source.vb

Would you like to take care of that with this PR @JordanDTaylor? The link I pasted is for the branch on this PR

JordanDTaylor committed:
Updates vb snippet

 Replaces deprecated reference in comments with ConfigurationManager.ConnectionStrings

dnfclas commented:
[![CLA assistant check](https://cla.dotnetfoundation.org/pull/badge/not_signed)](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=3939) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=3939) before we can accept your contribution.<br/><br/>:x: JordanDTaylor [sign now](https://cla.dotnetfoundation.org/dotnet/docs?pullRequest=3939)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/docs?pullRequest=3939) it.</sub>

JordanDTaylor commented:
Updated in the vb snippet as well.  Thanks for the insight. ☺

BillWagner commented:
I reviewed the VB changes before merging.  

Thanks again @JordanDTaylor We appreciate it.

I'll :shipit: now.  The changes should show up on the live site in the next day or two on our regular publishcing cycle.
